### ISSUE TITLE
About myown dataset

### ISSUE THREAD

**Issue by 1MiaoaXin** [AkonLau/DTRG#2]
Excuse, I use myown dataset to train the model, then use graph and ocl, while the dtrg_loss is keep on 0, how do you think of this issue?

**Comment by AkonLau** [comment]
I appreciate your interest in our work. Please make sure that the start_dtrg is set correctly. When the current epoch is less than the start_dtrg, the dtrg_loss would be kept on 0 for better iterating the class feature center matrix. For more details please refer to the paper about "Effects of starting epoch " in the ablation studies.

**Comment by 1MiaoaXin** [comment]
Thank you for your reply, but I have checked the parameters you mentioned and found that the settings are correct, but the dtrg loss is still 0

**Comment by AkonLau** [comment]
Please double-check that your input feature matrix is correct. Additionally, ensure that the hyperparameters "ocl" and "graph" are set to be true. Furthermore, it's important to try and have more than one instance for each class as much as possible.
